src/faas/start.ts

```diff
@@ -70,19 +70,13 @@ export async function start<Request = any, Response = any>(
       const nitricResponse = await func(nitricRequest);
 
       // Return parsed http response...
-      if (
-        nitricResponse &&
-        nitricResponse['status'] &&
-        nitricResponse['headers'] &&
-        nitricResponse['body']
-      ) {
+      if (nitricResponse instanceof NitricResponse) {
         const typedResponse = nitricResponse as NitricResponse<Response>;
 
-        res.writeHead(typedResponse.status);
-
         Object.keys(typedResponse.headers).forEach((k) => {
           res.setHeader(k, typedResponse.headers[k]);
         });
+
         send(res, typedResponse.status, typedResponse.body);
         return;
       } else if (!nitricResponse) {
```

Comment:
Shouldn't we handle the case where it's not an instance of NitricResponse, but it is a valid object that should be treated as the response body?

Reply:
This should already be the case:

See line 88. We only need to use send() in the case where we want to send custom headers or a custom response code.